DigiMeSDK/Core/DMEClient.m

```diff
@@ -119,6 +131,30 @@ - (void)authorizeWithCompletion:(nullable AuthorizationCompletionBlock)authoriza
     }];
 }
 
+#pragma mark - Postbox
+
+- (void)createPostbox
+{
+    
```

Comment:
This should totally invoke the main method below. 😆

Reply:
It probably should ;)